### ISSUE TITLE
SolidJs Support 

### ISSUE THREAD

**Issue by vincent-vade** [Aslemammad/vitext#8]
Hi,

Firstly this project look cool... 
It is possible to add Solid support  in the future ? 

Best regards

**Comment by Aslemammad** [comment]
Hello.
Solid is a perfect thing, I like the no-virtual-dom part, And I'd love to add it (like preact). But I'm not that familiar with its community & .... So could you elaborate on how we can do that & Why?

(I mention @ryansolid so he can give his thoughts)

**Comment by ryansolid** [comment]
Well Solid does not have React compat so this might be a challenge for any project not prepared to be multi-framework. It isn't a easy swap like Preact is.

**Comment by Aslemammad** [comment]
Yes, we can first bring simple solid ssr, then work on things like "vitext/head", "vitext/app" or "vitext/dynamic". How about that?

**Comment by PuruVJ** [comment]
@Aslemammad superb idea in my opinion. This could become the go to way to do opinionated React, Preact and Solid SSG/SSR apps

**Comment by Aslemammad** [comment]
@PuruVJ Exactly.
